analysis/src/ProcessCmt.ml

```diff
@@ -593,9 +593,11 @@ and forStructure ~name ~env strItems =
       strItems []
   in
   let attributes =
-    match strItems with
-    | {str_desc = Tstr_attribute attribute} :: _ -> [attribute]
-    | _ -> []
+    strItems
+    |> List.filter_map (fun (struc : Typedtree.structure_item) ->
+           match struc with
+           | {str_desc = Tstr_attribute attr} -> Some attr
+           | _ -> None)
```

Comment:
Get all attrs and no longer the first.


```sh
./analysis/rescript-editor-analysis.exe extractDocs ~/Desktop/projects/rescript-compiler/lib/ocaml/js.ml | head
```
Now we have the docstring.
```
{
  "name": "Js",
  "docstrings": ["The Js module mostly contains ReScript bindings to _standard JavaScript APIs_\n  like [console.log](https://developer.mozilla.org/en-US/docs/Web/API/Console
/log),\n  or the JavaScript\n  [String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String),\n  [Date](https://developer.mozilla.org/en-U
S/docs/Web/JavaScript/Reference/Global_Objects/Date), and\n  [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise)\n  classes.\n
\n  It is meant as a zero-abstraction interop layer and directly exposes JavaScript functions as they are. If you can find your API in this module, prefer this over an equiva
lent Belt helper. For example, prefer [Js.Array2](js/array-2) over [Belt.Array](belt/array)\n\n  ## Argument Order\n\n  For historical reasons, some APIs in the Js namespace 
(e.g. [Js.String](js/string)) are\n  using the data-last argument order whereas others (e.g. [Js.Date](js/date)) are using data-first.\n\n  For more information about these a
rgument orders and the trade-offs between them, see\n  [this blog post](https://www.javierchavarri.com/data-first-and-data-last-a-comparison/).\n\n  _Eventually, all modules 
in the Js namespace are going to be migrated to data-first though._\n\n  In the meantime, there are several options for dealing with the data-last APIs:\n\n  ```rescript\n  /
* Js.String (data-last API used with pipe last operator) */\n  Js.log(\"2019-11-10\" |> Js.String.split(\"-\"))\n  Js.log(\"ReScript\" |> Js.String.startsWith(\"Re\"))\n\n  /
* Js.String (data-last API used with pipe first operator) */\n  Js.log(\"2019-11-10\"->Js.String.split(\"-\", _))\n  Js.log(\"ReScript\"->Js.String.startsWith(\"Re\", _))\n\n
  /* Js.String (data-last API used without any piping) */\n  Js.log(Js.String.split(\"-\", \"2019-11-10\"))\n  Js.log(Js.String.startsWith(\"Re\", \"ReScript\"))\n  ```\n  ##
 Js.Xxx2 Modules\n\n  Prefer `Js.Array2` over `Js.Array`, `Js.String2` over `Js.String`, etc. The latters are old modules."],
  "items": [
  {
    "id": "Js.t",
    "kind": "type",
    "name": "t",
    "signature": "type t<'a> = 'a constraint 'a = {..}",
```

Reply:
No idea why just one attribute was picked up previously. Maybe @cristianoc knows?

Reply:
Not sure. I might be a good idea to run the tests and check that this does not change anything.

Reply:
The tests were not changed